src/main/java/core/basesyntax/serviceimpl/WriterServiceImpl.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax.serviceimpl;
+
+import core.basesyntax.service.WriterService;
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.List;
+
+public class WriterServiceImpl implements WriterService {
+    @Override
+    public void writeToFile(String filePath,List<String> lines) {
+        try (BufferedWriter writer = new BufferedWriter(new FileWriter(filePath))) {
+            writer.write("fruit,quantity");
+            writer.newLine();
+            for (String line: lines) {
```

Comment:
```suggestion
            for (String line : lines) {
```
don't forget to Ctrl Alt L regularly